### ISSUE TITLE
headers are omitted from the response when using this service and content delivery fails

### ISSUE THREAD

**Issue by mahpour** [Kentico/kontent-angular-http-service#2]
### Brief bug description

When plug in this service into DeliveryClient service app fails with the following error (both in browser and on SSR):

```
core.js:6237 ERROR TypeError: Cannot read property 'find' of undefined
    at getHasStaleContent (base-responses.js:18)
    at ListContentItemsResponse.BaseKontentResponseStandardDebug [as constructor] (base-responses.js:43)
    at new ListContentItemsResponse (responses.js:58)
    at MappingService.push../node_modules/@kentico/kontent-delivery/_commonjs/services/mapping.service.js.MappingService.listContentItemsResponse (mapping.service.js:96)
    at MapSubscriber.project (delivery-query.service.js:68)
    at MapSubscriber._next (map.js:29)
    at MapSubscriber.next (Subscriber.js:49)
    at CatchSubscriber._next (Subscriber.js:72)
    at CatchSubscriber.next (Subscriber.js:49)
    at CatchSubscriber._next (Subscriber.js:72)
```
### Expected behavior

Headers should be passed from http response into the DeliveryClient code

### Test environment

 - Platform/OS: Windows 10
 - Browser: Chrome 
 - Version: 81.0.4044.129

 - Angular setup (package.json):
```
.
.
.
"dependencies": {
    "@angular/animations": "9.1.4",
    "@angular/cdk": "^8.2.3",
    "@angular/common": "9.1.4",
    "@angular/compiler": "9.1.4",
    "@angular/core": "9.1.4",
    "@angular/flex-layout": "^9.0.0-beta.29",
    "@angular/forms": "9.1.4",
    "@angular/localize": "~9.1.4",
    "@angular/material": "^8.2.3",
    "@angular/platform-browser": "9.1.4",
    "@angular/platform-browser-dynamic": "9.1.4",
    "@angular/platform-server": "9.1.4",
    "@angular/router": "9.1.4",
    "@angular/service-worker": "~9.1.4",
    "@kentico/kontent-angular-http-service": "^4.0.0",
    "@kentico/kontent-core": "^6.0.0",
    "@kentico/kontent-delivery": "^9.1.1",
    "@microsoft/applicationinsights-web": "^2.5.3",
    "@ng-toolkit/universal": "^8.0.3",
    "@nguniversal/common": "9.1.0",
    "@nguniversal/express-engine": "9.1.0",
    "@ngx-translate/core": "^11.0.1",
    "@ngx-translate/http-loader": "^4.0.0",
    "applicationinsights": "^1.4.1",
    "buffer": "^5.4.2",
    "core-js": "^2.5.4",
    "domino": "^2.1.3",
    "express": "^4.15.2",
    "express-minify": "^1.0.0",
    "express-minify-html": "^0.12.0",
    "hammerjs": "^2.0.8",
    "helmet": "^3.21.2",
    "http-status-codes": "^1.3.2",
    "husky": "^2.1.0",
    "memory-cache": "^0.2.0",
    "ngx-cookie": "^4.1.2",
    "rxjs": "^6.5.3",
    "tslib": "^1.10.0",
    "zlib": "^1.0.5",
    "zone.js": "~0.10.2"
  },
  "devDependencies": {
    "@angular-devkit/build-angular": "~0.901.4",
    "@angular/cli": "^9.1.4",
    "@angular/compiler-cli": "~9.1.4",
    "@angular/language-service": "~9.1.4",
    "@kentico/kontent-cli": "0.0.8",
    "@kentico/kontent-management": "^0.3.19",
    "@kentico/kontent-model-generator": "^3.2.0",
    "@nguniversal/builders": "^9.1.0",
    "@types/express": "^4.17.4",
    "@types/hashids": "^1.0.30",
    "@types/helmet": "0.0.43",
    "@types/jasmine": "~2.8.8",
    "@types/jasminewd2": "~2.0.3",
    "@types/memory-cache": "^0.2.0",
    "@types/node": "^12.11.1",
    "codelyzer": "^5.1.2",
    "dotenv": "^8.2.0",
    "hashids": "^2.0.1",
    "jasmine-core": "~2.99.1",
    "jasmine-spec-reporter": "~4.2.1",
    "karma": "~4.0.0",
    "karma-chrome-launcher": "~2.2.0",
    "karma-coverage-istanbul-reporter": "~2.0.1",
    "karma-jasmine": "~1.1.2",
    "karma-jasmine-html-reporter": "^0.2.2",
    "mkdirp": "^0.5.1",
    "ngx-i18nsupport": "^0.17.1",
    "node-sass": "^4.13.0",
    "patch-package": "^6.1.2",
    "prettier": "1.17.0",
    "protractor": "~5.4.0",
    "rimraf": "^2.7.1",
    "ts-loader": "^5.2.0",
    "ts-node": "~7.0.0",
    "tslint": "~5.11.0",
    "tslint-config-prettier": "^1.18.0",
    "tslint-plugin-prettier": "^2.0.1",
    "typescript": "^3.8.3",
    "webpack-cli": "^3.1.0",
    "webpack-dev-server": "^3.8.0"
  }
.
.
.
```

### Additional context

This is happening when I attempted to upgrade from the old repo of this service to the new one. 

![kontent-angular-service](https://user-images.githubusercontent.com/202065/81249090-d1e8c180-8feb-11ea-81b3-cbbb9103b172.jpg)

`response.headers` is `undefined`

**Comment by mahpour** [comment]
I noticed `headers` is not defined in the following object construction:

https://github.com/Kentico/kontent-angular-http-service/blob/5e2a7b63ba1edd45bf95c8e8aadf9d599dd3ebda/lib/angular-http.service.ts#L87

**Comment by Enngage** [comment]
Hi,

Try using the newly released `4.1.0`. Should work there.

**Comment by mahpour** [comment]
Thanks! It’s resolved now
